### ISSUE TITLE
On Linux and MacOS, `khiops_env` Does Not Detect `mpiexec` in Conda-Based Contexts

### ISSUE THREAD

**Issue by popescu-v** [KhiopsML/khiops#476]
#### Description
Currently, on Linux and MacOS operating systems, `khiops_env` performs a `command -v mpiexec` test and sets `KHIOPS_MPI_COMMAND` if the test succeeds.
However, in Conda-based environments (e.g. globally-installed JupyterHub - see related Python library issue: https://github.com/KhiopsML/khiops-python/issues/283 ), `CONDA_PREFIX` is neither defined, nor in the `PATH`.
Hence, Conda environment-specific `mpiexec` is not detected and `KHIOPS_MPI_ERROR` is set to `"We didn't find mpiexec in the regular path. Parallel computation is unavailable: Khiops is launched in serial"`.

The output of `khiops_env --env` in such environments looks like:

```
KHIOPS_PATH /path/to/miniconda3/envs/test_10_2_3_1_rc_1/bin/MODL_mpich
KHIOPS_COCLUSTERING_PATH /path/to/miniconda3/envs/test_10_2_3_1_rc_1/bin/MODL_Coclustering
KHIOPS_MPI_COMMAND 
KHIOPS_JAVA_PATH 
KHIOPS_CLASSPATH 
KHIOPS_LAST_RUN_DIR /path/to/tmp
KHIOPS_PROC_NUMBER 32
KHIOPS_TMP_DIR 
KHIOPS_MEMORY_LIMIT 
KHIOPS_API_MODE 
KHIOPS_RAW_GUI 
KHIOPS_DRIVERS_PATH /path/to/miniconda3/envs/test_10_2_3_1_rc_1/lib
KHIOPS_JAVA_ERROR 
KHIOPS_MPI_ERROR We didn't find mpiexec in the regular path. Parallel computation is unavailable: Khiops is launched in serial
KHIOPS_MPI_VERBOSE true
```

#### Questions/Ideas
- The solution, on Linux and MacOS systems, is to fall back to checking `KHIOPS_MPI_COMMAND=$(dirname $KHIOPS_PATH)/mpiexec`
- On Windows systems, the issue should be moot, as `%_KHIOPS_HOME%` is used for setting `%KHIOPS_MPI_COMMAND%` in Conda environments, and `%_KHIOPS_HOME%` is set to the parent directory of `khiops_env.cmd`.

### Context
- Khiops version: 10.2.3
- Log file (use `khiops -e log.txt`): Any
- Scenario file (use `khiops -o scenario._kh`): Any
- OS description (use `khiops -s`): Ubuntu 22.04

**Comment by popescu-v** [comment]
Should also adapt CI additions from khiops-python, in order to test the Conda-based environments here as well. E.g. https://github.com/KhiopsML/khiops-python/blob/6488a1d95e6e66cb6c7198d8d104176cf9ac047a/.github/workflows/conda.yml#L111.

**Comment by popescu-v** [comment]
Closed by PR #479
